src/pages/index.tsx

```diff
@@ -3,34 +3,75 @@ import React from 'react';
 import Layout from '../components/layout/layout';
 import SEO from '../components/seo';
 import { Grid, GridItem } from '../components/grid/grid';
+import { Typography } from '../components/typography/typography';
+import styled from 'styled-components';
+import { up, down } from '../components/breakpoint/breakpoint';
+
+const Title = styled.div`
+  margin-top: 108px;
+  margin-bottom: 280px;
+
+  ${up('md')} {
+    margin-top: 192px;
+    margin-bottom: 360px;
+  }
+`;
+
+const Subtitle = styled.div`
+  margin-top: 80px;
+  margin-bottom: 80px;
+
+  ${up('md')} {
+    margin-top: 160px;
+    margin-bottom: 120px;
+  }
+`;
+
+const DesktopOnlyBreak = styled.br`
```

Comment:
I would remove this. We don't need responsive breaks. That's an antipattern. You can't control them if the text is coming from any external source anyway. This is just too much.

Reply:
Wanted to make our designers super happy :D But yeah, it also breaks accessibility so it doesn't work anyway.